src/git.ts

```diff
@@ -575,15 +593,55 @@ export function handleGitStatus(_actionData: ActionData): string | undefined {
             mergeChanges: repo.state.mergeChanges.map((change: Change) => translateChange(change)),
             HEAD: {
                 name: repo.state.HEAD?.name,
-                type: repo.state.HEAD !== undefined ? RefTypeStrings[repo.state.HEAD?.type] : undefined,
+                type: repo.state.HEAD !== undefined ? RefTypeStrings[repo.state.HEAD.type] : undefined,
                 ahead: repo.state.HEAD?.ahead,
                 behind: repo.state.HEAD?.behind,
                 commit: repo.state.HEAD?.commit,
                 remote: repo.state.HEAD?.remote,
                 upstream: repo.state.HEAD?.upstream,
             },
         };
-        NEURO.client?.sendContext(`Git status: ${JSON.stringify(state)}`);
+
+        // Helper function to map changes
+        function mapChanges(array: StateStringProps[], prefix?: string) {
+            const changes: string[] = [];
+            array.map((change: StateStringProps) => {
+                if (change.originalFileName && change.renamedFileName) {
+                    changes.push(`${prefix ? prefix : ''}(${change.status}) ${change.originalFileName} -> ${change.renamedFileName}\n`);
+                } else if (change.fileName) {
+                    changes.push(`${prefix ? prefix : ''}(${change.status}) ${change.fileName}\n`);
+                } else {
+                    changes.push(`${prefix ? prefix : ''}A(n) ${change.status} file had some missing data.`);
```

Comment:
- Simplify `undefined` checks
- Handle "a" / "an"
```suggestion
                    changes.push(`${prefix ?? ''}(${change.status}) ${change.originalFileName} -> ${change.renamedFileName}`);
                } else if (change.fileName) {
                    changes.push(`${prefix ?? ''}(${change.status}) ${change.fileName}`);
                } else {
                    const a = 'aeiou'.includes(change.status[0]) ? 'an' : 'a';
                    changes.push(`${prefix ?? ''}${a} ${change.status} file had some missing data.`);
```